src/lib/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ add_library(common-lib ${HDR_FILES} ${SRC_FILES})
 
 target_compile_options(common-lib PRIVATE -Wall -Wextra -pedantic -Wno-unused-parameter)
 
+target_include_directories(common-lib PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Could you explain this modification?

Reply:
It was done during a troubleshooting action but not deleted after. Sorry, it is just another mistake.